index.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+exports.getQueryString = function (maybeUrl) {
+	if (!maybeUrl) {
+		return;
+	}
+
+	var pos = maybeUrl.indexOf('?');
+	if (pos !== -1) {
+		return maybeUrl.substring(pos + 1);
+	}
+};
```

Comment:
Happy to make this change also. But be aware it means output is inconsistent for invalid input -empty string would return empty string instead of null.